src/MediaTypeBase.php

```diff
@@ -119,4 +120,14 @@ public function attachConstraints(MediaInterface $media) {}
   public function calculateDependencies() {
     return [];
   }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function validateConfigurationForm(array &$form, FormStateInterface $form_state) { }
```

Comment:
Should we also add default implementation for buildConfigurationForm()? There might be plugins that require no configuration. It is very unlikely though...